CONTRIBUTING.md

```diff
@@ -2,6 +2,30 @@
 
 Contributions welcome - just raise a PR and make sure the tests pass!
 
+## Testing
+
+**First make sure you have the following dependencies.**
+
+- [rbenv](https://github.com/rbenv/rbenv#installation)
+
+**Next install Ruby and the gems required for testing.**
+
+```sh
+rbenv install
+gem install bundler
```

Comment:
This step should be unnecessary given we're using Ruby >= 2.6

Reply:
Oh yes, I keep forgetting about that. Removed.